src/convertToObject.js

```diff
@@ -3,16 +3,34 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example in
+ *  [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ *  (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const convertStyles = sourceString.split(';').map(styles => {
+    return styles.trim().split(':').map(stylesElement => {
+      return stylesElement.trim();
+    }).join(':');
+  }).filter(elem => {
+    return elem !== '';
+  });
```

Comment:
![image](https://github.com/mate-academy/js_style-to-object/assets/59209337/87829615-3b14-4760-bc54-da5ab47c53a0)